src/test/resources/com/jcabi/w3c/invalid-html-response.xml-without-schema

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
@carlosmiranda name of this file is the main problem. Files with ".xml" extension are automatically validated during performing "qulice" profile and it brakes the build. This is why I changed the extension for now. Is there an other way to exclude one .xml file from those checking?